zproject/default_settings.py

```diff
@@ -442,5 +442,8 @@ class JwtAuthKey(TypedDict):
 # Use half of the available CPUs for data import purposes.
 DEFAULT_DATA_EXPORT_IMPORT_PARALLELISM = (len(os.sched_getaffinity(0)) // 2) or 1
 
-# How long after the last upgrade to nag users that the server is insecure
-SERVER_UPGRADE_NAG_DEADLINE = 365
+# How long after the last upgrade to nag users that the server needs
+# to be upgraded because of likely security releases in the meantime.
+# Default is 18 months, constructed as 12 months before someone should
+# upgrade, plus 6 months for the system administrator to get around to it.
+SERVER_UPGRADE_NAG_DEADLINE = 30 * 18
```

Comment:
Small thing before this setting is in a release so that it'd be a pain to change it: can we add `_DAYS` to the name?

In general when settings are in some particular units, I find it helpful to be explicit in the name what the units are. Especially when looking at code where the setting is used.

Reply:
Oh yeah, I was meaning to do that originally and then lost the thought when debugging something.